comment/api/serializers.py

```diff
@@ -2,13 +2,15 @@
 from django.utils import timezone
 
 from django.apps import apps
+from django.db.models import ImageField
 
 from rest_framework import serializers
 
 from comment.conf import settings
 from comment.models import Comment, Flag, Reaction
 from comment.utils import process_anonymous_commenting, get_user_for_request, get_profile_instance
 from comment.messages import EmailError
+from collections import OrderedDict
```

Comment:
This line should be at the top. Standard library imports should be placed at the top as per `PEP8`. You may read about the recommendations here: https://www.python.org/dev/peps/pep-0008/#imports